src/plot.jl

```diff
@@ -0,0 +1,163 @@
+export plot_single_flow, plot_graph, plot_assets_capacity
+
+"""
+    plot_single_flow(graph,          asset_from, asset_to, rp)
+    plot_single_flow(energy_problem, asset_from, asset_to, rp)
+
+Plot a single flow over a single representative period, given a graph or energy problem,
+the "from" (exporting) asset, the "to" (importing) asset, and the representative period.
+
+"""
+#TODO Rework using CairoMakie instead of Plots
+function plot_single_flow(graph::MetaGraph, asset_from::String, asset_to::String, rp::Int64)
+    rp_partition = graph[asset_from, asset_to].partitions[rp]
+    time_dimension = 1:length(rp_partition)
+    flow_value = [graph[asset_from, asset_to].flow[(rp, B)] for B in rp_partition]
+
+    Plots.plot(
+        time_dimension,
+        flow_value;
+        title = string("Flow from ", asset_from, " to ", asset_to, " for RP", rp),
+        titlefontsize = 10,
+        legend = false,
+    )
+end
+
+function plot_single_flow(
+    energy_problem::EnergyProblem,
+    asset_from::String,
+    asset_to::String,
+    rp::Int64,
+)
+    plot_single_flow(energy_problem.graph, asset_from, asset_to, rp)
+end
+
+"""
+    plot_final_flow_graph(graph)
+    plot_final_flow_graph(energy_problem)
+
+Given a graph or energy problem, plot the graph with the "final" (initial + investment) flow capacities,
+represented by the thickness of the graph edges, as well as displayed values.
+"""
+function plot_graph(graph::MetaGraph)
+    # Choose colors
+    nodelabelcolor = RGB(0.345, 0.164, 0.376)
+    transportflowcolor = RGB(1, 0.647, 0)
+    assetflowcolor = RGB(0.988, 0.525, 0.110)
+    nodecolor = RGBA(0.345, 0.164, 0.376, 0.5)
+
+    node_labels = labels(graph) |> collect
+
+    # Create a temporary graph that has arrows both directions for transport
+    temp_graph = DiGraph(nv(graph))
+    for e in edges(graph)
+        add_edge!(temp_graph, e.src, e.dst)
+        if graph[node_names[e.src], node_names[e.dst]].is_transport
+            add_edge!(temp_graph, e.dst, e.src)
+        end
+    end
+
+    # Calculate node size based on initial and investment capacity
+    node_size = []
+    node_names = []
+    for a in node_labels
+        node_total_capacity =
+            graph[a].initial_capacity +
+            graph[a].investable * graph[a].investment * graph[a].capacity
+        push!(node_names, "$a \n $(node_total_capacity)") # "Node Name, Capacity: ##"
+        push!(node_size, node_total_capacity)
+    end
+    node_size = node_size * 98 / maximum(node_size) .+ 5
+
+    # Calculate edge width (and set color) depending on type and (if transport) capacity
+    edge_colors = []
+    edge_width = []
+    edge_labels = []
+    for e in edges(temp_graph)
+        from = node_names[e.src]
+        to = node_names[e.dst]
```

Comment:
```suggestion
        from = node_labels[e.src]
        to = node_labels[e.dst]
```

Reply:
Ha sorry for my laziness. Should really be testing before pushing. :|